ts/Dashboards/Components/DataGridComponent/DataGridSyncs/DataGridVisibilitySync.ts

```diff
@@ -56,13 +56,10 @@ const syncPair: Sync.SyncPair = {
             }
 
             const columnId = cursor.column;
-            dataGrid.update({
-                columns: {
-                    [columnId]: {
-                        enabled: cursor.state !== 'series.hide' + groupKey
-                    }
-                }
-            });
+            dataGrid.updateColumns([{
```

Comment:
It is `async`. Shouldn't we await it?

Reply:
Maybe not await (we do it at the end of an event anyway), but good point, I'll leave `void` there for now.